tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -68,29 +89,29 @@ Verify odh-model-controller Deployment
     ${containerNames} =  Create List  manager
     Verify Deployment    ${odh_model_controller}  3  1  ${containerNames}
 
-Temporary Label MM Namespace
-    ${label} =    Run    oc label namespace ${MODEL_MESH_NAMESPACE} opendatahub.io/generated-namespace=true --overwrite=true
-    Log    ${label}
-    Run Keyword And Continue On Failure  Should Be Equal As Strings    ${label}    namespace/${MODEL_MESH_NAMESPACE} not labeled
-
 Verify Openvino Deployment
-    Run Keyword And Continue On Failure  Temporary Label MM Namespace
-    @{ovms} =  Oc Get    kind=Pod    namespace=${MODEL_MESH_NAMESPACE}    label_selector=name=modelmesh-serving-ovms-1.x
+    [Arguments]    ${project_name}=${PRJ_TITLE}    ${num_replicas}=1
+    @{ovms} =  Oc Get    kind=Pod    namespace=${project_name}   label_selector=name=modelmesh-serving-model-server-${project_name}
     ${containerNames} =  Create List  rest-proxy  oauth-proxy  ovms  ovms-adapter  mm
-    Verify Deployment    ${ovms}  2  5  ${containerNames}
-    ${all_ready} =    Run    oc get deployment -l name=modelmesh-serving-ovms-1.x | grep 2/2 -o
-    Should Be Equal As Strings    ${all_ready}    2/2
-
-Delete Model Serving Resources
-    [Documentation]    wrapper keyword that runs oc commands to delete model serving stuff
-    ...    Temporary until included in RHODS
-    ${output}=    Run    cd tests/Resources/Files && ./uninstall-model-serving.sh ${ODH_NAMESPACE} ${MODEL_MESH_NAMESPACE}
-    Log  ${output}
-    # Ignore all not found for delete commands, only output should be the not found of the patch command
-    Should Be Equal As Strings    Error from server (NotFound): kfdefs.kfdef.apps.kubeflow.org "odh-modelmesh" not found    ${output}
-
-Teardown Model Serving
-    [Documentation]  delete modelmesh stuff, Temporary until included in RHODS
-    # Seems like it's taking 10 minutes to stop reconciling deployments
-    Wait Until Keyword Succeeds  15 min  15s  Delete Model Serving Resources
-    Run    rm -rf modelmesh-serving
\ No newline at end of file
+    Verify Deployment    ${ovms}  ${num_replicas}  5  ${containerNames}
+    ${all_ready} =    Run    oc get deployment -n ${project_name} -l name=modelmesh-serving-model-server-${project_name} | grep ${num_replicas}/${num_replicas} -o
+    Should Be Equal As Strings    ${all_ready}    ${num_replicas}/${num_replicas}
```

Comment:
is it needed? if the previous command return something, than I guess this is automatically passed. Because in the previous "oc get.." you are looking explicitly for `${num_replicas}/${num_replicas}`

Reply:
The grep could return an empty line (if the pods are not replicas/replicas), so I need to check that what is returned is the correct thing